encoding/ipfix/element.go

```diff
@@ -4,63 +4,88 @@ import (
 	"fmt"
 )
 
+const (
+	fieldSpecifierSize            = 4
+	fieldSpecifierWEnterpriseSize = 8
+)
+
+// ErrorElement is an error definition associated with IPFIX Elements.
+type ElementError struct {
+	msg string
+	err error
+}
+
+// Associate Error() function to the ElementError.
+func (e *ElementError) Error() string {
+	return e.msg
+}
+
+// Define Unwrap function on the ElementError.
+func (e *ElementError) Unwrap() error { return e.err }
+
+// Then create an error object using MyError struct.
```

Comment:
Codacy found an issue: [comment on exported function ElementErrorInstance should be of the form "ElementErrorInstance ..."](https://app.codacy.com/manual/mateumann/clouseau/pullRequest?prid=6315366)